themes/default/content/blog/iac-recommended-practices-wrapping-up/index.md

```diff
@@ -0,0 +1,81 @@
+* [IaC Recommended Practices: Developer Stacks and Git Branches](/blog/iac-recommended-practices-developer-stacks-git-branches/)
+* [IaC Recommended Practices: Structuring Pulumi Projects](/blog/iac-recommended-practices-structuring-pulumi-projects/)
+* [IaC Recommended Practices: Using Stack References](/blog/iac-recommended-practices-using-stack-references/)
+* [IaC Recommended Practices: RBAC and Security](/blog/iac-recommended-practices-rbac-and-security/)
+* [IaC Recommended Practices: Using Automation API](/blog/iac-recommended-practices-using-automation-api/)
+* **IaC Recommended Practices: Wrapping Up** (this post)
+
+## Recapping infrastructure as code with Pulumi at Zephyr
+
+As this series has progressed, we've been showing you how Zephyr's use of Pulumi has changed in response to the organization, the team, and the application has changed. In case you haven't been following along---or in case you need a refresher---here's a brief summary of what's happened in the series:
+
+* Zephyr started with a single project in a single repository with two stacks, one for production and one for testing. _(Details are available in [the first blog post](/blog/iac-recommended-practices-code-organization-and-stacks/) in the series.)_
+* Zephyr quickly added per-developer stacks as a way to enhance developer productivity. _(You can read about per-developer stacks in [the second post](/blog/iac-recommended-practices-developer-stacks-git-branches/) in the series.)_
+* As Zephyr continued to grow, their single Pulumi project grew into three different Pulumi projects: one for base infrastructure, one for their Kubernetes platform, and one for the online store application. _(The [third post in the series](/blog/iac-recommended-practices-structuring-pulumi-projects/) describes the reasoning for adopting multiple projects.)_
+* Accompanying the switch to multiple projects, Zephyr implemented stack references (to pass information between stacks in different projects) and applied role-based access control (RBAC) to the stacks in Pulumi Cloud. _(Refer back to [the fourth post](/blog/iac-recommended-practices-using-stack-references/) and [the fifth post](/blog/iac-recommended-practices-rbac-and-security/) in the series for more details on each of these areas.)_
+* Zephyr realized a need to split out their data layer, going from three projects to four projects, and along the way saw an opportunity to add a higher level of orchestration using the Pulumi Automation API. _(Refer back to [our sixth post](/blog/iac-recommended-practices-using-automation-api/) for more details on Zephyr's initial use of Automation API.)_
+
+Since the last blog post, the Zephyr team has continued to explore Automation API, and has added an Automation API program that allows their developers to deploy the Zephyr online store directly from the associated GitHub repositories---developers don't even need to clone down the repositories first.
```

Comment:
```suggestion
Since the last blog post, the Zephyr team has continued to explore [Automation API](/docs/using-pulumi/automation-api/), and has added an Automation API program that allows their developers to deploy the Zephyr online store directly from the associated GitHub repositories---developers don't even need to clone down the repositories first.
```

Reply:
Great suggestion, fixed